src/components/tag/tag.js

```diff
@@ -0,0 +1,97 @@
+import React, { PureComponent } from 'react';
+import Proptypes from 'prop-types';
+import cx from 'classnames';
+
+import closeIcon from './assets/legend-close.svg';
+import styles from './tag-styles.scss';
+import Icon from '../icon/icon';
+
+class Tag extends PureComponent {
+  handleClick = e => {
+    const { label, onRemove } = this.props;
+    e.preventDefault();
+    onRemove(label);
+  };
+
+  render() {
+    const { data, theme, color, label, canRemove, tooltipId } = this.props;
+
+    const tagContent = (
+      <React.Fragment>
+        <span
+          className={cx(styles.dot, theme.dot)}
+          style={{ backgroundColor: color }}
+        />
+        {
+          data && data.title && tooltipId
```

Comment:
There is missing the [react-tooltip](https://github.com/wwayne/react-tooltip) dependency.
As it's optional maybe doesn't worth it to include where the component is used but would be nice to include it here and explain [how it works](https://github.com/wwayne/react-tooltip#using-react-component-as-tooltip)